src/otx/cli/install.py

```diff
@@ -0,0 +1,127 @@
+"""OTX CLI Installation."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Need to fix to new style

Reply:
Thank you. I will fix this.

Reply:
Thanks, Done - https://github.com/openvinotoolkit/training_extensions/pull/2679/commits/f210aa31cef980728345aa4007aa6a0cf01ba3dc